### DISCUSSION TITLE
Modifying mod packs

### DISCUSSION THREAD

**Discussion by IniJamiel** [itzg/docker-minecraft-server#2268] [Q&A] [answered]
I'm currently running Cities: Skylands modpacks through Auto Curse Forge. I'm using Portainer to run my dockers in VPS. Is there a way to remove and add mods from the server? I tried to delete the mods manually. But it seems that it just redownload the file.

**Comment by itzg** [comment, marked as answer]
The docs for AUTO_CURSEFORGE describe how you can exclude mods, so that'll handle "removing" them. As for adding mods, you can augment with any of the other mechanisms described at https://docker-minecraft-server.readthedocs.io/mods-and-plugins/

**Reply by IniJamiel** [reply]
Okay, will try. Thanks a lot

**Comment by KSEGIT** [comment]
Is it possible to exclude mods from modrinth pack?

**Reply by itzg** [reply]
https://docker-minecraft-server.readthedocs.io/en/latest/types-and-platforms/mod-platforms/modrinth-modpacks/#excluding-files

**Reply by KSEGIT** [reply]
Thank you!
